checker/build.gradle

```diff
@@ -374,6 +374,7 @@ test {
         excludeCategories 'org.checkerframework.checker.test.junit.ainferrunners.AinferTestCheckerStubsTest'
         excludeCategories 'org.checkerframework.checker.test.junit.ainferrunners.AinferTestCheckerAjavaTest'
         excludeCategories 'org.checkerframework.checker.test.junit.ainferrunners.AinferNullnessJaifsTest'
+        excludeCategories 'org.checkerframework.checker.test.junit.ainferrunners.AinferNullnessAjavaTest'
```

Comment:
This should also be added to the list of `ainferTest` tasks around line 840, so that it does get run.

Reply:
Is that in addition to this change?